### PR TITLE
Pad least significant bits of trace ID

### PR DESCRIPTION
**Pull request by codeboten**
Moved padding from left to right. Also ensuring the TraceID is not modified as it gets propagated if it started out as 128-bit

### CODE REVIEW

File: tracer_transport_test.go [hunk 1]
```diff
@@ -592,7 +593,7 @@ var _ = Describe("Tracer Transports", func() {
 				Expect(context).To(BeEquivalentTo(knownContext3))
 			})
 
-			It("should drop higher 64 bits of a 128 bit TraceID", func() {
+			It("should error with an invalid TraceID", func() {
```
Comment (iredelmeier) [inline tracer_transport_test.go]: Nit: Might be adding a couple words in the description about what makes this ID invalid. At a glance, it looks like it should be fine, and I think is actually the same ID used in a "valid" test above.

File: raw_span.go [hunk 2]
```diff
@@ -38,6 +38,9 @@ type SpanContext struct {
 	// A probabilistically unique identifier for a [multi-span] trace.
 	TraceID uint64
 
+	// Least significant bits of TraceID
+	TraceIDLower uint64
```
Comment (iredelmeier) [inline raw_span.go]: Do we need to add the field yet, or can we just drop the lower bits for now? Doesn't look like we have anything consuming the new `TraceIDLower`, e.g., for propagation.

File: tracer_transport_test.go (hunk 1 above)
Comment (codeboten) [reply tracer_transport_test.go]: I'll update it, the ID is 31 characters instead of 32 as the one above.

File: raw_span.go (hunk 2 above)
Comment (codeboten) [reply raw_span.go]: The field gets propagated is used in the b3 propagation: https://github.com/lightstep/lightstep-tracer-go/pull/232/files#diff-c7e8819bc83013a318340945ad302ef3R36

Comment (iredelmeier) [reply raw_span.go]: Doh!

**Pull request merged**